### ISSUE TITLE
Will you release the training code?

### ISSUE THREAD

**Issue by gsygsy96** [xuxy09/RSC-Net#3]
Hi @xuxy09, thanks for your code. Applying contrastive learning in human mesh recovery is interesting. Do you plan to release the implementation of this part? Thanks again.

**Comment by xuxy09** [comment]
Updated. Please find the loss here: https://github.com/xuxy09/RSC-Net/blob/dbab8b3527293aa19413a0214546215738ea4a40/train/trainer.py#L33

**Comment by gsygsy96** [comment]
Thanks!
